addons/point_of_sale/tools/posbox/configuration/load_certificate.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Write the server configuration
+function load_certificate () {
+	CRT="${1}"
```

Comment:
rename all those files, it is really weird that they are named 'ginx-selfsigned.*' and they don't contains selfsigned certificates